MonoGame.Framework/Input/MouseState.cs

```diff
@@ -29,25 +29,62 @@ public struct MouseState
         /// <param name="xButton1">XBUTTON1's state.</param>
         /// <param name="xButton2">XBUTTON2's state.</param>
         /// <remarks>Normally <see cref="Mouse.GetState()"/> should be used to get mouse current state. The constructor is provided for simulating mouse input.</remarks>
-		public MouseState (
-			int x,
-			int y,
-			int scrollWheel,
-			ButtonState leftButton,
-			ButtonState middleButton,
-			ButtonState rightButton,
-			ButtonState xButton1,
-			ButtonState xButton2)
-		{
-			_x = x;
-			_y = y;
-			_scrollWheelValue = scrollWheel;
-			_leftButton = leftButton;
-			_middleButton = middleButton;
-			_rightButton = rightButton;
+        public MouseState(
+            int x,
+            int y,
+            int scrollWheel,
+            ButtonState leftButton,
+            ButtonState middleButton,
+            ButtonState rightButton,
+            ButtonState xButton1,
+            ButtonState xButton2) : this(
+                x,
+                y,
+                scrollWheel,
+                leftButton,
+                middleButton,
+                rightButton,
+                xButton1,
+                xButton2,
+                horizontalScrollWheel: 0
+            )
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the MouseState.
+        /// </summary>
+        /// <param name="x">Horizontal position of the mouse in relation to the window.</param>
+        /// <param name="y">Vertical position of the mouse in relation to the window.</param>
+        /// <param name="scrollWheel">Mouse scroll wheel's value.</param>
+        /// <param name="leftButton">Left mouse button's state.</param>
+        /// <param name="middleButton">Middle mouse button's state.</param>
+        /// <param name="rightButton">Right mouse button's state.</param>
+        /// <param name="xButton1">XBUTTON1's state.</param>
+        /// <param name="xButton2">XBUTTON2's state.</param>
+        /// <param name="horizontalScrollWheel">Mouse horizontal scroll wheel's value.</param>
+        /// <remarks>Normally <see cref="Mouse.GetState()"/> should be used to get mouse current state. The constructor is provided for simulating mouse input.</remarks>
+        public MouseState(
+            int x,
+            int y,
+            int scrollWheel,
+            ButtonState leftButton,
+            ButtonState middleButton,
+            ButtonState rightButton,
+            ButtonState xButton1,
+            ButtonState xButton2,
+            int horizontalScrollWheel)
+        {
+            _x = x;
+            _y = y;
+            _scrollWheelValue = scrollWheel;
+            _leftButton = leftButton;
+            _middleButton = middleButton;
+            _rightButton = rightButton;
             _xButton1 = xButton1;
-            _xButton2 = xButton2;
-		}
+            _xButton2 = xButton2;
+            HorizontalScrollWheelValue = horizontalScrollWheel;
```

Comment:
Remain consistent with all the other properties and implement as an explicit backing field.